docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  app:
+    image: friendsofsymfony/fos-http-cache
```

Comment:
It's also can be a pain for contributing to have it local as if the way of installing varnish has changed (archive not being available, new dependancy to add for compiling, other stuff...) then people wanting to test on local will need to have the knowledge of installing varnish. 

Having it on the hub remove the necessity of this knowledge and ensure that the image you pull is working.

IMO the best would be to have it on local by default, but still publish it to the docker hub, so if someone struggle when building he can still use the last version published.